resource/policies/base/match_op.h

```diff
@@ -6,6 +6,7 @@ typedef enum match_op_t {
     MATCH_ALLOCATE,
     MATCH_ALLOCATE_W_SATISFIABILITY,
     MATCH_ALLOCATE_ORELSE_RESERVE,
+    MATCH_GROW_ALLOCATION,
```

Comment:
Suggestion for naming consistency:

```suggestion
    MATCH_GROW_ALLOCATE,
```
I shouldn't be reviewing this early, I've read that word less than 10 times and am getting semantic satiation. "ALLOCATE" looks like some kind of fruit name to me right now.

Reply:
I waffled on this and am back to preferring `MATCH_GROW_ALLOCATION`. My current thinking is that the pattern is verb_verb_{noun/verb/conjunction}, and `GROW_ALLOCATION` is clearer.